release_docs/RELEASE.txt

```diff
@@ -289,11 +279,42 @@ Bug Fixes since HDF5-1.12.2 release
       (H5Pset_libver_bounds(..., H5F_LIBVER_EARLIEST, H5F_LIBVER_V18)). This
       has now been fixed.
 
+    - Fixed an assertion in a previous fix for CVE-2016-4332
+
+      An assert could fail when processing corrupt files that have invalid
+      shared message flags (as in CVE-2016-4332).
+
+      The assert statement in question has been replaced with pointer checks
+      that don't raise errors. Since the function is in cleanup code, we do
+      our best to close and free things, even when presented with partially
+      initialized structs.
+
+      Fixes CVE-2016-4332 and HDFFV-9950 (confirmed via the cve_hdf5 repo)
+
+    - Fixed segfault on file close
+
```

Comment:
This entry is brought from HDF5 1.10.11 but is new to HDF5 1.12.3.  Was the corresponding code fix included in hdf5_1_12?

Reply:
Might have been accidental when moving the previous item?

Reply:
No, it was just moved up from line 387, which I didn't notice before.  It belongs in the file;  I'm not sure why the merge moved it.